"""
exceptions.py: Exceptions common for all all tests
"""


class DockerException(Exception):
    pass
